docs/guidelines.md

```diff
@@ -223,3 +223,37 @@ Because `window` is both the global object and represents the browser window, th
 Use the most customary reference in each case.
 If you're not sure what's customary, look to high-profile published examples, such as those on MDN reference pages for the feature.
 ([#913](https://github.com/web-platform-dx/web-features/pull/913#discussion_r1572601975))
+
+## `caniuse` values
+
+The `caniuse` key references one or more [Can I Use](https://caniuse.com/) feature IDs.
+This is the part of a Can I Use URL after `https://caniuse.com/`.
+For example, the Can I Use feature ID for [Flexbox](https://caniuse.com/flexbox) is `flexbox`.
```

Comment:
How about "A list of the `canisuse` keys also be found in the `caniuse` repo in the [`features-json` folder](https://github.com/Fyrd/caniuse/tree/main/features-json)."

Reply:
Good idea, But there are some weird exceptions, so I can do one better:

```suggestion
For example, the Can I Use feature ID for [Flexbox](https://caniuse.com/flexbox) is `flexbox`.
For a complete list of IDs, run `npx tsx ./scripts/caniuse.ts`.
```